README.md

```diff
@@ -158,6 +158,23 @@ The environment variables used by the frontend are also checked for type safety.
 3. Import it in the file where it's used with `import { env } from "@/env";` and use it like `env.MY_ENV_VAR`. At this point, your environment variable should be working locally.
 4. To ensure it also works in CircleCI and Silta, also add it to`.circleci/config.yml` and `silta-next.yml`.
 
+### Testing with Cypress
+
+The template includes example tests to be run with Cypress
+
+#### Running tests locally
+
+To run the tests locally, first start the frontend and backend servers with `lando start`, then run `lando npm run cypress:run` to start the Cypress test runner.
```

Comment:
Hmm, when I run `lando npm run cypress:run` I get this error message. Any ideas what I'm doing wrong? (I did indeed run `lando start` prior to this.)

```
> next4-drupal-template-by-wunder@0.1.0 cypress:run
> cypress run

It looks like this is your first time using Cypress: 12.14.0


Your system is missing the dependency: Xvfb

Install Xvfb and run Cypress again.

Read our documentation on dependencies for more information:

https://on.cypress.io/required-dependencies

If you are using Docker, we provide containers with all required dependencies installed.

----------

Error: spawn Xvfb ENOENT

----------

Platform: linux-arm64 (Debian - 10.13)
Cypress Version: 12.14.0
```

Reply:
By the way, even if this did work, you also have to start the frontend web server with `lando npm run build && lando npm run start` OR `lando npm run dev`, right?